tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+import os
+
+import pytest
+
+from src.python_x509_pkcs11.pkcs11_handle import PKCS11Session
+
+ON_GITHUB = bool(os.getenv("GITHUB_ACTIONS", False))
```

Comment:
Does this variable already exists in a github pipeline, or do we need to add it?

Reply:
`GITHUB_ACTIONS` is always set to `true` while running under github actions.